### PR TITLE
feat: docs for fms v2

### PR DESCRIPTION
**Pull request by 2hwk**
## Summary

Documentation for FMS v2
 * Removing cFMS v1.5 specific errata and information.

### Location

Discord username (if different from GitHub): 2Cas

### CODE REVIEW

**Review by frankkopp — APPROVED** [review]

**Review by Sleinmaster — APPROVED** [review]

**Comment by tracernz** [comment]
Please note: circling approaches will no longer be available in the FMS, in line with the findings in https://discord.com/channels/738864299392630914/748096886531162123/1229357652182695956

**Comment by Valastiri** [comment]
For tracking purposes:
- https://github.com/flybywiresim/aircraft/pull/7692

File: docs/pilots-corner/advanced-guides/flight-planning/disco.md [hunk 1]
```diff
@@ -89,26 +89,16 @@ If the aircraft is flying into a MANUAL leg, NAV mode remains engaged and predic
...
-    simulation of the Honeywell FMS.
-
-!!! warning "STAR and Approach Discontinuity - Inaccuracy"
-    If your STAR contains other waypoints after the IAF (initial approach fix) that you have selected via an approach transition (VIA), the FMS will not automatically connect the STAR to the approach at the IAF.
-
-    This is a small problem with our current implementation. It will be corrected when we update to version 2 (fms-v2) of our implementation, which contains an even better 
-    simulation of the Honeywell FMS.
-
 
+    Even if the waypoints form a direct line, it's not completely safe to remove a discontinuity unless you understand the above points
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/flight-planning/disco.md]:
```suggestion
    Even if the waypoints form a direct line, it's not completely safe to remove a discontinuity unless you understand the above points.
```

**Comment by Valastiri** [comment]
PR may require additional input to support FMSv2 implementation.

**Pull request merged**